draft-irtf-cfrg-opaque.md

```diff
@@ -371,20 +371,32 @@ OPAQUE relies on the following protocols and primitives:
 Note that we only need the base mode variant (as opposed to the verifiable mode
 variant) of the OPRF described in {{I-D.irtf-cfrg-voprf}}.
 
-- Cryptographic hash function:
-  - Hash(m): Compute the cryptographic hash of input message `m`. The type of the
-    hash is determined by the chosen OPRF group.
-  - Nh: The output size of the Hash function.
-
-- Authenticated Key Exchange (AKE, {{instantiations}}):
-  - Npk: The size of the public keys used for the key exchange protocol.
-  - Nsk: The size of the private keys used for the key exchange protocol.
+- Key Derivation Function (KDF):
+  - Extract(salt, ikm): Extract a pseudorandom key of fixed length `Nl` bytes from
+    input keying material `ikm` and an optional byte string `salt`.
+  - Expand(prk, info, L): Expand a pseudorandom key `prk` using optional string `info`
+    into `L` bytes of output keying material.
+  - Nl: The output size of the `Extract()` function in bytes.
```

Comment:
This isn't really important, but if we name `Nh` after the hash length and `Nm` for the mac length, we might as well call this one `Ne` for the `Extract()` output length.

Reply:
I considered that, but `Ne` conflicts with the VOPRF terminology for "size of Element", so I went with `Nl`.

Reply:
What if the VOPRF terminology changes to `Np` for "size of Point"?

Reply:
Point implies curve, which may not always be the case. :)

Reply:
Resurrecting this, at the risk of being annoying, but what do you think about `Nx`?

Reply:
Why that over `Nl`?

Reply:
isn't the second letter referring to the function name? if _e_ is already taken, _x_ refers more easily to "extract" than _l_

Reply:
This is a pretty epic bike shed! I picked `l` for length. I do not care what we go with here.